gigareview/console/tests/functional/EMReportJobCest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace console\tests\functional;
+use console\tests\FunctionalTester;
+use yii\console\ExitCode;
+use console\models\ManuscriptsWorker;
+
+class EMReportJobCest
+{
+    public function _before(FunctionalTester $I)
+    {
+        $I->runShellCommand("./yii_test migrate/down all --interactive=0", false);
+        $I->runShellCommand("./yii_test migrate/up --interactive=0", false);
+    }
```

Comment:
Another way of resetting the `reviewdb_test` database is to use the [Db module](https://codeception.com/docs/modules/Db) with a `pgdmp` file.

Reply:
It requires writing a script to make `reviewdb_test.pgdmp` file, and  including `migrate/up`, `migrate/down` in `_before()` seems to be a simpler fix.